packages/terra-demographics-banner/src/selectable-demographics-banner/SelectableDemographicsBanner.module.scss

```diff
@@ -0,0 +1,45 @@
+:local {
+  .background-tile {
+    background-color: var(--terra-demographics-banner-background-color, #004c76);
```

Comment:
Theres really just one themeable variable for this?

Reply:
Most of the styles have been moved to DemographicsBanner.module.css. Additional themeable variables have been added here.